language/de-DE/joomla.ini

```
@@ -327,13 +335,13 @@ JGRID_HEADING_ORDERING_DESC="Sortierung absteigend"
 ; if there is an error connecting database before initialisation, en-GB.lib_joomla.ini can't be loaded
 ; we therefore have to load the strings from en-GB.ini
 
-JLIB_DATABASE_ERROR_ADAPTER_MYSQL="Der MySQL-Adapter „mysql“ ist nicht verfügbar!"
-JLIB_DATABASE_ERROR_ADAPTER_MYSQLI="Der MySQL-Adapter „mysqli“ ist nicht verfügbar!"
-JLIB_DATABASE_ERROR_CONNECT_DATABASE="Verbindung zur Datenbank „%s“ ist nicht möglich"
-JLIB_DATABASE_ERROR_CONNECT_MYSQL="Es konnte keine Verbindung zu MySQL aufgebaut werden!"
-JLIB_DATABASE_ERROR_DATABASE_CONNECT="Keine Verbindung zur Datenbank!"
+JLIB_DATABASE_ERROR_ADAPTER_MYSQL="Der MySQL-Adapter „mysql“ ist nicht verfügbar."
+JLIB_DATABASE_ERROR_ADAPTER_MYSQLI="Der MySQL-Adapter „mysqli“ ist nicht verfügbar."
+JLIB_DATABASE_ERROR_CONNECT_DATABASE="Verbindung zur Datenbank \"%s\" ist nicht möglich"
```

```suggestion
JLIB_DATABASE_ERROR_CONNECT_DATABASE="Verbindung zur Datenbank „%s“ ist nicht möglich"
```
Bitte voerst so lassen.